conan/tools/files/files.py

```diff
@@ -185,13 +185,34 @@ def rename(conanfile, src, dst):
             raise ConanException("rename {} to {} failed: {}".format(src, dst, err))
 
 
-def load_build_json(conanfile):
-    path = os.path.join(conanfile.generators_folder, CONAN_TOOLCHAIN_ARGS_FILE)
-    contents = load(conanfile, path)
-    data = json.loads(contents)
-    return data
+def load_toolchain_args(generators_folder=None):
+    """
+    Helper function to load the content of any CONAN_TOOLCHAIN_ARGS_FILE
+
+    :param generators_folder: `str` folder where is located the CONAN_TOOLCHAIN_ARGS_FILE.
+    :return: <class 'configparser.SectionProxy'>
+    """
+    args_file = os.path.join(generators_folder, CONAN_TOOLCHAIN_ARGS_FILE) if generators_folder \
+        else CONAN_TOOLCHAIN_ARGS_FILE
+    if os.path.exists(args_file):
```

Comment:
Shouldn't it fail if it doesn't exist? I think it would be safer.

Reply:
I'm rethinking it and I'm not completely sure about it. Now, some tests are failing because it's not supposed to be any `conanbuild.conf` created... Are we sure that we want that behavior then? @lasote let us know your opinion, please

Reply:
Yes, I would say it has to exist. Some build helpers need more that file than others but in general, if they read the file, the file should be there.